sdcm/prometheus.py

```diff
@@ -200,6 +200,39 @@ def run(self):
             if delta > 0:
                 time.sleep(int(delta))
 
+    def silence(self, alert_name: str, duration: int = None, start: datetime.datetime = None, end: datetime.datetime = None) -> None:
+        """
+        Silence an alert for a duration of time
+
+        :param alert_name: name of the alert as it configured in prometheus
+        :param duration: duration time in seconds, if None, start and end must be defined
```

Comment:
couldn't it be like, when i start a disruption i set it to silence, and when it is done i set it to un-silence?

Reply:
We might, but we need to know when it is safe to remove the silence, it might not be right after the nemesis is done, but a minute or so later

Reply:
sure, so in that case, once i unset silent, we could give an extra minute before really removing the silence...

Reply:
Now we have this options in the `EventsFilter`